spec/defines/sensu_handler_spec.rb

```diff
@@ -177,4 +177,9 @@
     it { should contain_sensu_handler('myhandler').with_handle_flapping( true ) }
   end
 
+  context 'handle_silenced' do
```

Comment:
I've added another spec to confirm that `handle_silenced` when it's left as the default of `false`